src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -57,26 +64,75 @@ public static void main(String[] args) {
   @PostMapping("/")
   public void onEvent(@RequestBody JsonNode event) throws IOException, GeneralSecurityException {
     switch (event.at("/type").asText()) {
+
       case "ADDED_TO_SPACE":
         String spaceType = event.at("/space/type").asText();
         if ("ROOM".equals(spaceType)) {
           String displayName = event.at("/space/displayName").asText();
           replyText = String.format("Thanks for adding me to %s. Type \"@MacroBot help\" at any time to see my instructions.", displayName);
+          helpMessage = "To use a room member's macro, the creator of a macro must write \"@MacroBot initiate MacroName \". " + 
+            "Once this has been sent, any room member can use the initialized macro by writing \"@MacroBot MacroName <your message> \" " + 
+            "If your macro has already been used in a thread, you may omit the MacroName and can simply write \"@MacroBot <your message>\".";
+
```

Comment:
Could we make this help message a private static final constant?

Reply:
also in our demo - probably important to have screenshots illustrating the space initalization step and the thread step as two separate things. Could be confusing to a new user, and if we make screenshots of the UI I think people will understand it better

Reply:
Oh you're right, thanks, it can be final. I've made this update.

Reply:
helpMessage should be a local variable (in the method) whenever you set it it should be something like:
helpMessage = ADD_TO_SPACE_MESSAGE;
where ADD_TO_SPACE_MESSAGE is a constant defined at the class level.

Reply:
same about replyText btw.